syntax = "proto3";

import "api/chat/message.proto";

package api.chat.request;

message SendMessage {
    uint64 chat_id = 1;
    api.chat.Message message = 2;
}
